modules/lightning_features/lightning_layout/modules/lightning_inline_block/lightning_inline_block.links.contextual.yml

```diff
@@ -0,0 +1,4 @@
+inline_block_content.block_edit:
+  title: 'Edit'
```

Comment:
There must be at least one contextual link present in order for Quick Edit to work.